docs/install-and-setup.md

```diff
@@ -66,6 +84,7 @@ You can also install `jj` via [MacPorts](https://www.macports.org) (as
 the `jujutsu` port):
 
 ```shell
+# Installs the latest release
 sudo port install jujutsu
 ```
 
```

Comment:
The "(port page)" on the line below looked a bit odd to me. What do you think about replacing the `MacPorts` link above by it? We probably don't need to link both to MacPorts and the specific port page.